code/game/machinery/doors/airlock.dm

```diff
@@ -1091,7 +1091,7 @@
 		if(obj_flags & EMAGGED)
 			return FALSE
 		use_power(50)
-		playsound(src.loc, doorOpen, 30, 1)
+		playsound(src.loc, doorOpen, 30)
```

Comment:
just src

also why are you removing the frequency variance?

Reply:
the open sound sounds terrible with pitch change.